tool/tsh/common/tsh_test.go

```diff
@@ -3555,22 +3555,28 @@ func mockSSOLogin(t *testing.T, authServer *auth.Server, user types.User) client
 	return func(ctx context.Context, connectorID string, priv *keys.PrivateKey, protocol string) (*auth.SSHLoginResponse, error) {
 		// generate certificates for our user
 		clusterName, err := authServer.GetClusterName()
-		require.NoError(t, err)
+		assert.NoError(t, err)
```

Comment:
Won't changing this to an assert cause a potential panic on "clusterName.GetClusterName()" ?

Reply:
Good catch, I'll `assert.Fail` and return the error to caller.